transport/owr_transport_agent.c

```diff
@@ -1158,11 +1158,12 @@ static GstElement *add_dtls_srtp_bin(OwrTransportAgent *transport_agent, guint s
     if (!is_encoder) {
         g_object_get(session, "dtls-certificate", &cert, NULL);
         g_object_get(session, "dtls-key", &key, NULL);
-
+        GST_CAT_INFO_OBJECT(_owrsession_debug, session, "transport_agent cert = %s\nkey = %s\n", cert, key);
```

Comment:
I don't think that is necessary, I'll remove this line. Is there anywhere else you see an actual (non debug or error) printout?